crates/but-db/migrations/2025-10-15-105125_update_worktrees_schema/up.sql

```diff
@@ -0,0 +1,15 @@
+-- Update worktrees table to new schema
+-- SQLite doesn't support ALTER COLUMN, so we need to recreate the table
+
+-- Create new table with updated schema
+CREATE TABLE `worktrees_new`(
+	`path` BLOB NOT NULL PRIMARY KEY,
+	`base` TEXT NOT NULL,
+	`created_from_ref` BLOB
```

Comment:
These two items can be stored directly in the worktree, so the database isn't needed for these write-once items.

These items would just be standard Git refs that can be stored per worktree, for instance in `.git/worktrees/<name>/BASE|CREATED_FROM`.